pkg/service/manifest/storage/storage.go

```diff
@@ -30,6 +31,8 @@ type StoredManifest struct {
 
 type StoredApplication struct {
 	ID             string                         `json:"id"`
+	Status         credential.Status              `json:"status"`
+	Reason         string                         `json:"reason"`
```

Comment:
should this be an enum or user defined?

Reply:
I'm imagining it's user defined free form text. We can change later once we see real use cases.